var/spack/repos/builtin/packages/py-h5py/package.py

```diff
@@ -47,11 +47,11 @@ class PyH5py(PythonPackage):
 
     # Build dependencies
     # h5py@3.11 can build with cython@3.x
-    depends_on("py-cython@0.29.31:", type="build", when="@3.11:")
-    depends_on("py-cython@0.29.31:0", type="build", when="@3.9:")
+    depends_on("py-cython@0.29.31:3", type="build", when="@3.11:")
```

Comment:
Remind me, the `:3` here means something like `<4`, right?

Reply:
Yes (see https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges).